files/sdw-admin.py

```diff
@@ -139,11 +167,31 @@ def main():
         sys.exit(0)
     args = parse_args()
     if args.validate:
-        print("Validating...")
+        print("Validating...", end="")
         validate_config(SCRIPTS_PATH)
+        print("OK")
     elif args.apply:
+        print(
+            "SecureDrop Workstation should be installed on a fresh Qubes OS install.\n"
+            "The installation process will overwrite any user modifications to the\n"
+            f"{BASE_TEMPLATE} TemplateVM, and will disable old-format qubes-rpc\n"
+            "policy directives.\n"
```

Comment:
I don't think this latter part is true anymore - I think we have decided that we won't disable them, we'll just make sure all our rules take precedence before the old-style rules can even load.

Reply:
sdw-admin will get another pass to clean up output and add more stringent prod checks, we can bake that final decision in (and maybe check the rpc rules config as well as checking for VMs) when we do.